examples/official-site/sqlpage/migrations/01_documentation.sql

```diff
@@ -211,7 +211,23 @@ INSERT INTO component(name, icon, description) VALUES
     'The value entered by the user in a field named x will be accessible to the target SQL page as a variable named $x.
     For instance, you can create a SQL page named "create_user.sql" that would contain "INSERT INTO users(name) VALUES($name)"
     and a form with its action property set to "create_user.sql" that would contain a field named "name".');
+INSERT INTO parameter(component, name, description_md, type, top_level, optional) SELECT 'form', * FROM (VALUES
+    -- top level
+    ('enctype', '
+When ``method="post"``, this specifies how the form-data should be encoded
+when submitting it to the server.
```

Comment:
I think just this line is enough. Advanced users will find it when they need it, and we wont confuse others with things they don't need.

Reply:
Sorry, I don't follow. There is only one line here.

Reply:
I was saying that this line is fine, and we don't need the other changes in the documentation besides it.

Reply:
Please could you make the desired changes? I find the documentation of the row level param and the side-effect of file type input helpful so I'm loathe to remove it.